src/test/java/net/sf/jabref/importer/fileformat/BibTeXMLImporterTest.java

```diff
@@ -53,21 +54,28 @@ public void testGetItemsEmpty() {
 
     @Test
     public void testGetFormatName() {
-        BibTeXMLImporter importer = new BibTeXMLImporter();
         Assert.assertEquals("BibTeXML", importer.getFormatName());
     }
 
     @Test
     public void testGetCLIId() {
-        BibTeXMLImporter importer = new BibTeXMLImporter();
         Assert.assertEquals("bibtexml", importer.getId());
     }
 
     @Test
-    public void testIsRecognizedFormatReject() throws IOException {
-        BibTeXMLImporter importer = new BibTeXMLImporter();
+    public void testsGetExtensions() {
```

Comment:
I like the bottomline `Test everything that could possibly break` of http://programmers.stackexchange.com/a/130926. 
But since these tests are already in place, we can leave them there.